contrib/check-image-size.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import sys
+from enum import Enum
+
+# Enum Class for checking image size
+class ImageSizeCheck(Enum):
+    OK = "OK"
+    TOO_BIG = "TOO_BIG"
+    IGNORED = "IGNORED"
+    UNKNOWN = "UNKNOWN"
+
+
+# Some devices pad their images to IMAGE_SIZE and apply a firmware header.
+# Exclude this from the image size check.
+excluded_devices = [
+    "tplink_cpe210-v1",
+    "tplink_cpe210-v2",
+    "tplink_cpe210-v3",
+    "tplink_cpe220-v3",
+    "tplink_cpe510-v1",
+    "tplink_cpe510-v2",
+    "tplink_cpe510-v3",
+    "tplink_cpe710-v1",
+    "tplink_wbs210-v1",
+    "tplink_wbs210-v2",
+    "tplink_wbs510-v1"
+]
+
+
+def open_json(file_path):
+    with open(file_path, 'r') as f:
+        return json.load(f)
+
+
+def load_openwrt_profile_json(json_path):
+    profiles = []
+    profile_json = open_json(json_path)
+    for profile_name, profile_data in profile_json["profiles"].items():
+        device_profile = {
+            "name": profile_name,
+        }
+        if "image_size" in profile_data.get("file_size_limits", {}):
```

Comment:
```suggestion
        if "image" in profile_data.get("file_size_limits", {}):
```
Otherwise this never matches